src/pages/selfhosted/self-hosted-vs-cloud-netbird.mdx

```diff
@@ -0,0 +1,50 @@
+# Self-hosted vs. Cloud-hosted NetBird
+
+NetBird can be used in two ways: self-hosted and cloud-hosted (SaaS). The self-hosted version is a standalone version
+that can be installed on your own servers while the cloud-hosted version is a SaaS (Software as a Service) that is hosted
+on our servers.
+
+Often times, the question arises: what is the difference between the two? This post gives a short overview of the
+differences between the two versions.
+
+## Installation and Maintenance
+
+NetBird platform is a combination of [multiple components](/about-netbird/how-netbird-works) that make it possible for
+machines to establish direct point-to-point connections and for network administrators to manage the network,
+e.g., control network access.
+
+When running the self-hosted version, you are responsible for installing and maintaining all the components as well as backing up
+and securing the data.
+The cloud-hosted NetBird only requires you to install the client software (NetBird agent) on your machines and log them in to the network.
+The cloud-hosted version is more suitable for organizations that want a hassle-free solution that is easy to set up and maintain.
+
+## Features
+
+While the self-hosted and cloud-hosted versions share the same core connectivity features, the cloud-hosted version has
+some additional features that are targeted at business customers and help with network automation and management. These features include:
+
+- **[Event streaming](https://docs.netbird.io/how-to/activity-event-streaming)** to 3rd parties.
+- **[EDR integrations](https://docs.netbird.io/how-to/endpoint-detection-and-response)** (e.g., CrowdStrike).
+- **[Identity provider synchronization](https://docs.netbird.io/how-to/idp-sync)** of users and groups.
+- **[Peer approval](/docs.netbird.io/how-to/approve-peers)**.
+- **[User invites](/how-to/add-users-to-your-network#direct-user-invites)**.
```

Comment:
Should this be consistent with the other URLs? Either all relative or all absolute